app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java

```diff
@@ -224,34 +214,36 @@ public int getItemViewType(int position) {
     }
 
     public class SubscriptionViewHolder extends RecyclerView.ViewHolder {
-        private final TextView feedTitle;
-        private final ImageView imageView;
-        private final TriangleLabelView count;
+        private final TextView title;
+        private final ImageView coverImage;
+        private final TextView count;
+        private final TextView fallbackTitle;
         private final FrameLayout selectView;
         private final CheckBox selectCheckbox;
+        private final CardView card;
+        private final Activity activity;
```

Comment:
This is not a static child class. You can simply access the parent class' member of the activity.

Reply:
Removed new variable to access parent class’ as requested